pkg/cable/metrics.go

```diff
@@ -55,32 +60,126 @@ var (
 			cableDriverLabel,
 			localClusterLabel,
 			localHostnameLabel,
+			localEndpointIpLabel,
+			remoteClusterLabel,
+			remoteHostnameLabel,
+			remoteEndpointIpLabel,
+		},
+	)
+	connectionStatusGauge = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "connection_status",
+			Help: "Number of connections and corresponding status (by cable driver and cable)",
+		},
+		[]string{
+			cableDriverLabel,
+			localClusterLabel,
+			localHostnameLabel,
+			localEndpointIpLabel,
+			remoteClusterLabel,
+			remoteHostnameLabel,
+			remoteEndpointIpLabel,
+		},
```

Comment:
Shouldn’t this have a label for the status too?

Reply:
then what will be the gauge's value?

Reply:
The number of connections per status, same as the operator.